doc/examples/dual_arms/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 add_subdirectory(dual_arm_panda_moveit_config)
 
-install(DIRECTORY launch
+install(DIRECTORY
   DESTINATION share/${PROJECT_NAME}
 )
```

Comment:
Thanks for noticing that this launch file was unused!

I believe you can delete this line completely now:

```suggestion
```